src/components/application_manager/src/policies/policy_handler.cc

```diff
@@ -964,8 +981,8 @@ void PolicyHandler::OnPendingPermissionChange(
     return;
   }
 
-  AppPermissions permissions =
-      policy_manager_->GetAppPermissionsChanges(policy_app_id);
+  AppPermissions permissions = CallPolicyManagerFunction(
```

Comment:
@SNiukalov Const?

Reply:
@AByzhynar 
Fixed in: https://github.com/smartdevicelink/sdl_core/pull/2614/commits/98dc17dda0378940c016a6b9a16f1e3992ee8e65